source/instance.txt

```diff
@@ -11,7 +11,16 @@ Compass Home
    :class: singlecol
 
 The Compass Home screen provides details regarding the
-MongoDB instance to which Compass is connected, including:
+MongoDB instances to which Compass is connected. 
+
+.. note::
+
+   Starting in version 1.44, you can connect to multiple MongoDB
+   instances at once through Compass. When you activate multiple
+   connections, Compass displays each active connection in a separate
+   tab. 
+
+The Compass Home screen includes the following information:
 
 - The connection name if the connection is a favorite connection,
```

Comment:
I think this is out of date--won't the name be the connection string? You can also save a connection as a favorite without specifying a unique name now

Reply:
Updated accordingly!